mesecons_commandblock/init.lua

```diff
@@ -156,6 +157,11 @@ local function commandblock_action_on(pos, node)
 			minetest.chat_send_player(owner, "The command "..cmd.." does not exist")
 			return
 		end
+		if #param > param_maxlen then
+			minetest.chat_send_player(owner, "Command parameters can only be " ..
+				param_maxlen .. " bytes long")
```

Comment:
Common sense sould tell you that `n < limit` is acceptable. I don't trust common sense, thus this suggestion:
```suggestion
			minetest.chat_send_player(owner, "Command parameters are limited to max. " ..
				param_maxlen .. " bytes.")
```